web/scripts/vite-utils.js

```diff
@@ -116,7 +116,8 @@ const BaseConfig = () => ({
 
   test: {
     root: ROOT,
-    environment: "happy-dom",
+    isolate: false,
```

Comment:
Why is `isolate: false` here?

Reply:
It's a new option. If `isolate` is true, every test will be ran in an isolated environment. This adds some startup delay, among other things. I don't think we need this feature, as I'm not leaving garbage everywhere when I'm writing tests. At least for now, I mean, it could be needed later, but for right now keeping it off gives us a performance boost.

This will be in the Vitest docs eventually, things are a bit WIP over there lol